clients/fides-js/src/components/Tcf/TcfPurposes.tsx

```diff
@@ -0,0 +1,45 @@
+import { h } from "preact";
+import { NoticeToggle } from "../NoticeToggles";
+import { PrivacyNotice } from "../../lib/consent-types";
+import FilterButtons from "./FilterButtons";
+import CookiesTable from "./CookiesTable";
+
+const TcfPurposes = ({ notices }: { notices: Array<PrivacyNotice> }) => {
+  const handleToggle = () => {};
+  return (
+    <div>
+      <FilterButtons />
+      <NoticeToggle notice={notices[0]} checked onToggle={handleToggle}>
+        <div style={{ padding: "0.5em" }}>
+          <NoticeToggle
+            notice={notices[1]}
```

Comment:
why is this using `notices[1]` and the overall `NoticeToggle` component using `notices[0]`?

Reply:
ah this was purely for demo purposes, so that the screenshot could show different notices 👍